### PR TITLE
support to skip crew ending sessions automatically

### LINKED ISSUE #198: Create the option for Crews to not automatically end_session() after kickoff
## 🚀 Feature Request

**💡 Got a brilliant idea?**
If you run `crew.kickoff()` multiple times during the same session, a `kickoff()` success will automatically run `agentops.end_session('Success')`. This means the developer loses control of their session.

**🎉 Describe the solution you'd like**
Update `crew.kickoff()` so that it doesn't automatically `end_session`. Or make it optional somehow.

### PR DESCRIPTION
**Pull request by braelyn-ai**
Closes #198 

Crew has the ability to automatically detect when all tasks are complete and to end execution. If the developer doesn't wish for their AgentOps session to end at this time, they can pass an optional parameter into `init()` to skip this.

### CODE REVIEW

File: agentops/client.py [hunk 1]
```diff
@@ -374,8 +378,12 @@ def end_session(
             end_state (str): The final state of the session. Options: Success, Fail, or Indeterminate.
             end_state_reason (str, optional): The reason for ending the session.
             video (str, optional): The video screen recording of the session
+            is_auto_end (bool, optional): is this an automatic use of end_session and should be skipped with skip_auto_end_session
         """
 
+        if is_auto_end and self.config.skip_auto_end_session:
```
Comment (HowieG) [inline agentops/client.py:384]: So in crew you're calling end_session(is_auto_end=True) ?

**Review by HowieG — APPROVED** [review]

**Comment by HowieG** [comment]
Approving but I think our init params are getting too long and gut tells me we could do this cleaner. Discussed w @siyangqiu and we're all gonna discuss moving over to a config object pattern

File: agentops/client.py (hunk 1 above)
Comment (braelyn-ai) [reply agentops/client.py:384]: correct

**Comment by braelyn-ai** [comment]
> Approving but I think our init params are getting too long and gut tells me we could do this cleaner. Discussed w @siyangqiu and we're all gonna discuss moving over to a config object pattern

I would agree with this, but I think it merits more discussion unrelated to this PR

**Pull request merged**